### ISSUE TITLE
[Discussion] Do we need the RAML Console?

### ISSUE THREAD

**Issue by sichvoge** [mulesoft/api-workbench#39]
Hi guys, really philosophical question. With the "RAML Editor Tools” feature that is more rich  and seriously better for development, do we even need the console inside the workbench. I am not using it anymore at all. We could even introduce the “TryIt” into the Editor tools. (edited)

I would like to have a conversation about this as I am seeing two things doing the same.

**Comment by blakeembrey** [comment]
@sichvoge I agree. There's some work needed on both sides of the experience, but merging this into a single tool should be a priority. Having both open during development doesn't make sense. I can see the value in features of each, but separate tools still don't make sense.

I'd love to see the editor tools tidied up so they are more user friendly and pull in some of the features the console implements, then do away with the console side of things.

**Comment by sichvoge** [comment]
I love the "RAML Editor Tools", I am using it more than I ever have the console. Especially with a certain level of modularization it get's very handy. I agree it needs some improvements.

**Comment by ddenisenko** [comment]
Well, we need the console for at least two very important reasons.

The first one - it is the only place at the moment, which purpose is to show user how methods and resource are going to finally look, after applying resource types, traits etc etc.

The second one is that at some day it will probably be at least an equivalent or better then current industry-standard consoles like Swagger one. Our current implementation is very young, its potential is great.

In example, it would be really nice and very doable to have a mock-server being able to serve requests basing on our examples, and then make our console to actually send requests to that mock server, so user can not only see the results of RAML definitions, but also play a bit with service calls.
